\documentclass[a4paper,11pt]{report}
\usepackage{setspace}
\usepackage{fullpage}
\usepackage[square,sort&compress]{natbib}
%\usepackage[authoryear,sort&compress]{natbib}
\usepackage{tabu, longtable}
\usepackage{pdflscape}
\usepackage{bibentry}
\usepackage[table]{xcolor}
\usepackage{tablefootnote}
\usepackage{hyperref}
\usepackage{amsmath}
\usepackage{graphicx}
\usepackage{amssymb}\usepackage[T1, OT1]{fontenc}
\usepackage{hyperref}
\usepackage{subcaption}
\usepackage{titlesec}
\renewcommand{\dh}{\fontencoding{T1}\selectfont{\symbol{240}}}
\graphicspath{ {../pics/} }




%MACROS

%footnotes in tables
%\makesavenoteenv{tabular}

% shade rows in table
%\rowcolors{2}{gray!15}{white}

% alternate rowcolors for all tables
\let\oldtabular\tabular
\let\endoldtabular\endtabular
\renewenvironment{tabular}{\rowcolors{2}{white}{gray!15}\oldtabular}{\endoldtabular}

% alternate rowcolors for all long-tables
\let\oldlongtable\longtable
\let\endoldlongtable\endlongtable\renewenvironment{longtable}{\rowcolors{2}{white}{gray!15}\oldlongtable} {
\endoldlongtable}

% allow column widths in tables with alignment
\newcolumntype{L}[1]{>{\raggedright\let\newline\\\arraybackslash\hspace{0pt}}m{#1}}
\newcolumntype{C}[1]{>{\centering\let\newline\\\arraybackslash\hspace{0pt}}m{#1}}
\newcolumntype{R}[1]{>{\raggedleft\let\newline\\\arraybackslash\hspace{0pt}}m{#1}}



\titleformat{\chapter}[display]
{\normalfont\huge\bfseries}{\chaptertitlename\ \thechapter}{20pt}{\Huge}
\titlespacing*{\chapter}{0pt}{0pt}{30pt}


\usepackage{parskip}

\title{\huge \textbf{Using three dimensional chromatin structure to understand autoimmune disease mechanisms} \\
\textbf{} \\
First Year Report}
\author{ \LARGE
 \textbf{Oliver S Burren}\\ \\
\LARGE \url{ob219@cam.ac.uk}\\ \\
\LARGE under the supervision of Chris Wallace \\ \\
}
\date{\huge \today}


% define the title
%\author{Oliver S Burren}
%\title{Using }
\begin{document}

\begin{figure}
\centering
\includegraphics[width=0.3\linewidth]{UniCam.pdf}
\end{figure}




% generates the title
\maketitle

\thispagestyle{empty} 
\vspace*{\stretch{1}}
\begin{small}
This report is submitted as part requirement for the  PhD Programme in Medical Science at Cambridge University. Much of the work described in this report is collaborative. My specific contributions  include the assembly of a collection of GWAS summary statistics, the  development, implementation of \textit{blockshifter} and COGS algorithms and interpretation of subsequent analysis.
The report may be freely copied and distributed provided the source is explicitly acknowledged.
\end{small}
\vspace*{\stretch{1}}

\clearpage

\singlespacing

\begin{abstract}
 Genome wide association studies (GWAS) have uncovered hundreds of genetic regions that are responsible for human susceptibility to autoimmune disease.  Identification and functional characterisation of causal variation, a pre-requisite for therapeutic intervention, has proved difficult. This challenge stems not only from the underlying genetic architectures that are often refractory to the statistical resolution of causal variation, but also, due to complex and tissue specific three dimensional (3D) chromatin organisation, the cryptic nature of the genes that such variation targets.  Recent empirical developments have enabled high resolution maps of 3D chromatin organisation to be elucidated using promoter capture Hi-C (PCHi-C) that might allow the physical linkage of causal variants with target  genes. The aim of my PhD is to develop statistical methods to integrate GWAS with these tissue specific PCHi-C maps and other genomic data in order to better understand the biology of autoimmune susceptibility.  
 
 I developed \textit{blockshifter}, a  competitive circularised permutation method for examining promoter interacting regions (PIRs) for trait associated variant enrichment, that allows for correlation between variants and interactions. By applying \textit{blockshifter} to publicly available summary statistics for eight autoimmune and 23 non-autoimmune traits, and PCHi-C maps for 17 haematopoetic tissues I found enrichment for autoimmune variants in lymphoid tissues, which was strongest in activated CD4$^+$ T cells. Next, I developed a Bayesian method, COGS, to generate PCHi-C supported gene scores to prioritise variants, genes and tissue contexts for functional validation.  I applied COGS to prioritise 2,604 genes across all 31 traits and tissues. Qualitatively, COGS gene scores were more specific and sensitive when compared to similar methods using either proximal gene variants or topologically associated domain (TAD) boundaries. I found that genes prioritised by COGS were significantly enriched for genes differentially expressed between healthy and diseased immune subsets ($P = 0.002$ - ulcerative colitis). In contrast, scores for TAD and proximal methods showed no enrichment. 
 
I applied COGS to a set of six autoimmune traits for which dense fine mapping summary statistics were available in \url{http://www.immunobase.org}. To assess the effect of multiple proximal, but independent variants, on scores, I modified COGS to accept marginal posterior probabilities computed using GUESSFM,  for four traits, for which full genotype data was available. Due to \textit{blockshifter} results  I limited initial analysis to PCHi-C maps for activated and non-activated CD4$^{+}$ T cells, prioritising 256 genes. One of these, \textit{IL2RA}, was prioritised in CD4$^+$ T cells across multiple autoimmune diseases. Allele specific expression analysis of rs61839660 by a collaborator, provided support for a PIR in intron 1  modulating expression of \textit{IL2RA}, in unactivated but not activated contexts.
 
In future work I will: a) complete detailed tissue specific analysis across all 17 tissues, b) explore approaches to the integration of other relevant genomic data sets, c) use simulated GWAS statistics to better characterise COGS scores, d) develop tissue specific gene set enrichment methods based on COGS scores.  
\end{abstract}


% insert the table of contents
\onehalfspacing
\tableofcontents
%\clearpage
%\printglossary[style=index,type=\acronymtype,nonumberlist]
\clearpage


\chapter{Introduction}
\section{Motivation}

A key property of the adaptive immune system is the ability to recognise pathogens from self-antigens.  Dysregulation of this process results in damage to healthy tissues and autoimmunity. Currently, over 80 diseases have been found to have an underlying autoimmune pathogenesis, with approximately half presenting as rare diseases~\citep{HayterCook2012}. The collective health burden of more common autoimmune diseases such as type 1 diabetes (T1D), rheumatoid arthritis (RA), inflammatory bowel disease (IBD) and multiple sclerosis (MS) is high with approximately 7 - 9$\%$ of the European population affected~\citep{CooperBynumSomers2009}. Although environment is a contributing factor in disease susceptibility, the genetic heritability, defined as the proportion of phenotypic variance attributable to genetic variability, is also important, ranging from 0.39 in primary biliary cirrhosis (PBC) to 0.9 in ankylosing spondylitis (AS)~\citep{Gutierrez-ArcelusRichRaychaudhuri2016}.  

Genome wide association studies (GWAS) have been instrumental in understanding the complex genetic architecture underlying autoimmune disease susceptibility with at least 324 distinct genetic loci robustly associated with one or more autoimmune diseases (\url{http://www.immunobase.org}, accessed $01/08/2016$). Focus is now drawn to elucidating the mechanisms by which underlying causal variation modulates phenotypic endpoints, a prerequisite for successful therapeutic development. Interestingly, the majority of associated variants fall outside of genes ~\citep{MauranoHumbertRynesEtAl2012} and the integrative analysis of chromatin marks with GWAS highlights a tissue specific specific regulatory role ~\citep{FarhMarsonZhuEtAl2015}.  Recent studies ~\citep{ClaussnitzerDankelKimEtAl2015,SmemoTenaKimEtAl2014,Davison2012-zk} have shown that such regulatory variants might, through chromatin conformation, regulate distal genes. However, further progress in this area has been hampered by incomplete knowledge of causal variants and their target genes and the specific tissue contexts in which they operate~\citep{Albert2015-jn}.

%% should I add in stuff about Martin and Dryden studies ?

To date systematic methods for incorporating physical interactions between variants and their target genes have not been attempted. In this report I describe tools and analytical methods to integrate genetic and high resolution promoter capture Hi-C (PCHi-C) data sets. I focus method development on using GWAS summary statistics for input, as due to legitimate privacy concerns, access to raw genotype data is more restrictive, limiting the number of traits that can be analysed. These methods provide a data driven approach to prioritising the putative causal variants underlying autoimmune disease susceptibility, and  relevant tissue contexts, genes and biological pathways within which they operate. I apply these methods to integrate 17 PCHi-C maps for primary human cells with GWAS and fine mapping data across 31 traits. Based on these results I extend this analysis to examine summary ImmunoChip dense mapping summary statistics for ten autoimmune traits in the context of CD4${^+}$ T cell activation.

\section{Fine mapping}
\label{sect:finemapping}
One of the first steps of this project is to identify optimal methods for fine mapping, that balance data availability, resolution and computational efficiency. Fine mapping is the process of refining association signals in a genomic region in order to characterise fine scale genetic architecture, a necessary step in order to identify putative causal variants. Progress in this area is challenging, due to the presence of linkage disequilibrium (LD), which in many cases means that the causal variation cannot be resolved statistically with current sample sizes~\citep{LiKellis2016}. 

 One approach, under the simplifying assumption that a single causal variant explains the effect within a given genetic region, is to apply Wakefield's synthesis of approximate Bayes Factors (ABF)~\citep{Wakefield2009} to each variant within the region. These can be converted into posterior probabilities for a SNP to be causal using methods described in ~\citet{The_Wellcome_Trust_Case_Control_Consortium2012-ad}. A sizeable benefit of this approach is that it circumvents privacy issues, as the input is limited to GWAS summary statistics rather than access to full genotyping data.

Wakefield defines the ABF as

\begin{equation}
	ABF =  \sqrt{1 - r} \times \exp{\left[\frac{Z^{2}}{2} \times r\right]}
\end{equation}

If a given set of summary statistics include $\hat{\beta}$, an  estimate of the $\log(\text{Odds Ratio})$ and $\sqrt{V}$, the standard error of the Odds Ratio for a variant, then we can compute  $Z= \frac{\hat{\beta}}{\sqrt{V}}$. Alternatively, if only univariate $p$-values are supplied a $Z$ score can be estimated using an inverse normal cumulative distribution function. If $r$, a shrinkage factor, is defined as the  the ratio of the prior variance on $\hat{\beta}$ to the total variance, then, $r = \frac{W}{V + W}$. $V$ is approximated using the variant minor allele frequency (MAF) and study sample size such that $V=\frac{1}{2Nf(1-f)}$ for quantitative traits, where $N$ is number of samples and $f$ is the MAF. In the case/control setting $V=\frac{1}{2Nf(1-f)s(1-s)}$ where $s$ is the proportion of cases. The value of $\sqrt{W}$, the standard deviation of a normal prior for $\beta$, depends on study design considerations. In this report we use values of 0.15 and 0.2 for  case/control  and quantitative trait settings  respectively, as discussed by \citet{GiambartolomeiVukcevicSchadtEtAl2014}.

Given the ABF for all SNPs in a given genetic block and letting $\pi_{i}$ be our prior probability that SNP $i$ is  causal for a trait, we can estimate the posterior probability that a SNP $i$  is causal using the following. 
% add equation here with an explanation
\begin{equation}
\label{form:bf_derivation}
\begin{split}
	PP_{i}& = P(\text{SNP}_{i}\text{ causal} | D)\\
	& = \frac{P(D | \text{SNP}_{i} \text{ causal})\pi_{i}}{\sum_{j}P(D | \text{SNP}_{j}\text{causal})\pi_{j} + P(D | H_{0})\pi_0}\\
	& = \frac{
				\frac{
					P(D | \text{SNP}_{i}\text{ causal})
				}
				{
					P(D|H_{0})
				}\pi_{i}
		}
		{
				\sum_{j=1}^n\frac{
					P(D | \text{SNP}_{j}\text{ causal})
				}
				{
					P(D|H_{0})
				}\pi_{j}  + \pi_0
		}	\\
	& = \frac{
		\text{BF}_{i}\pi_{i}
	}
	{
	(\sum_{j=1}^n\text{BF}_{j}\pi_{j}) + \pi_{0}
	} \\	
	\\
	& \approx \frac{
		\text{BF}_{i}\pi_{i}
	}
	{
	(\sum_{j=1}^n\text{BF}_{j}\pi_{j}) + 1
	} \\
\end{split}
\end{equation}
Note that as $\pi_{0} = 1-\sum_{j}\pi_{j} \approx 1$. 



However, current sample sizes are under powered and therefore have limited resolution, which has driven the the development of integrative techniques that couple fine mapping methods with functional information, such as DNase accessibility to further prioritise causal variants. Examples include \textit{fgwas}~\citep{Pickrell2014-xs}, PAINTOR~\citep{KichaevYangLindstromEtAl2014}, RiVIERA~\citep{LiKellis2016}   and the integration of eQTL data sets using Mendelian randomisation~\citep{ZhuZhangHuEtAl2016}. As an example ~\citet{Pickrell2014-xs} developed a Bayesian hierarchical framework implemented in \textit{fgwas} that estimates variable priors for each SNP based on other sources of annotation. For a given SNP $i$, in region $j$, the prior probability for association, $\pi_{ij}$, varies depending on the enrichment of annotations, $\lambda_{l}$. 

\begin{equation}
	\pi_{ij} = \frac{e^{x_{i}}}{\sum_{p \in S_j}e^{x_{p}}}
	\label{eqn:fgwas_var_prior}
\end{equation}

where $x_i$ is the sum of the effect of all the annotations that the $i^{th}$ SNP overlaps as shown below.  Here $\lambda{i}$ is the effect of annotation $l$ and $I_{il}$ is and indicator function as to whether SNP $i$ overlaps annotation $l$.  

\begin{equation}
	x_{i} = \sum_{l=1}^{L_{2}} \lambda_{l}I_{il}
	\label{eqn:fgwas_lambda}
\end{equation}

$\lambda_{l}$ is itself of interest as it indicates whether a given annotation is enriched for association with a trait of interest, discussed further in Section \ref{sect:vsea}. When combined with equation \ref{form:bf_derivation}, $\pi_{ij}$ allows the incorporation of functional information with genetic data allowing a modest increase in resolution and a resultant decrease in the number of causal SNPs to be considered~\citep{Pickrell2014-xs}. Whilst promising \textit{fgwas} requires careful cross-validation to prevent over fitting whereby the derived model is biased for the training set but performs poorly on a test set of data, and all such methods are limited by the input annotations available.

Where raw genotyping data for a study is available then stepwise regression could be used to fine map, however whilst this approach is attractive computationally, doubts exist over its validity~\citep{miller-1984}.  Having selected a single variant that best explains the variance of a trait, stepwise regression  looks for other variants that explain additional trait variance conditional on this `top' variant. This is not equivalent to explaining which variants jointly explain the variance of a trait. However, approaches that search the model space exhaustively are only computationally feasible for simple models incorporating a limited number of variants. An alternative approach is to use Monte Carlo methods to sample the model space allowing the consideration of multiple causal variants within a genetic region. One example is GUESSFM that uses a Bayesian evolutionary stochastic search algorithm to effectively sample the model space, and has been shown to have consistently better performance when variants are highly correlated~\cite{WallaceCutlerPontikosEtAl2015}. 

\section{Variant set enrichment}
\label{sect:vsea}
The next part of my project investigates whether the putative causal SNPs identified can be used to better understand the biology of autoimmune disease.  If these variants  are enriched for a particular annotation or gene set then this can provide global information on mechanisms, relevant tissue contexts and biological pathways. However this is complicated by both correlation between variants (due to linkage disequilibrium) and annotations. 

\begin{equation}
	\text{Var}(A + B) = \text{Var}(A) + \text{Var}(B) + 2\text{Cov}(A,B)
\end{equation}

If $A$ and $B$ are non independent variables then the covariance term $2\text{Cov}(A,B)$ can be almost as large as $\text{Var}(A) + \text{Var}(B)$. This causes the observed variance to be inflated compared to the theoretical variance if independence is assumed. Unfortunately most classical statistical tests based on linear sums of variables assume independence and thus the inflated variance of the computed test statistic increases type 1 error, that is the erroneous rejection of the null hypothesis. Empirical methods can be used to better estimate this variance. There are three broad sets of methods. Firstly, permuting case/control status and rerunning the original GWAS analysis, however this is very computationally expensive and requires access to the raw data~\citep{Evangelou2014-ed}. A second approach, is to use a suitable reference genotype set to compute covariance matrices, allowing the estimation of test statistic variance under the null hypothesis using the multivariate normal~\citep{LiuMcRaeNyholtEtAl2010,Burren2014-vh}. This approach is attractive as it obviates the need for raw genotyping data, however it is computationally expensive and scales exponentially with SNP density. To overcome this an LD pruning strategy is employed, resulting a reduction in resolution,  limiting application to larger genomic annotations such as genes. The third approach is to permute the annotations over the SNPs whilst maintaining the spatial correlation structure of the target annotation to estimate the variance under the null. The latter approach is attractive as it favours high resolution annotations and is computationally more tractable. GOSHIFTER is a recent implementation using a circular permutation strategy ~\citep{Trynka2015-wz} that demonstrates this approach finding enrichment for H3K4me3 marks in CD4$^{+}$ memory T cells. 

As previously mentioned in Section \ref{sect:finemapping}, integrative approaches such as \textit{fgwas} combine variant set enrichment with fine mapping strategies to simultaneously prioritise annotations and variants. This approach additionally benefits from jointly assessing multiple annotations, thus if there is additional correlation between different annotation types this can be adjusted for.  It should be noted that all methods are dependent on the quality and coverage of input annotations. A recent study fine mapping IBD causal variants found that 21 variants with extremely high probability ($>$ 95$\%$) to be causal did not overlap any functional annotations, drawing attention to the inadequacies of current genomic functional annotation~\citep{Huang2015-ug}.

A major set of limitations inherent in the above analyses stems from the methods employed to organise genes into sets categorised by some functional annotation. Publicly available pathway databases include Gene Ontology~\citep{AshburnerBallBlakeEtAl2000}, KEGG~\citep{KanehisaGoto2000} and Reactome~\citep{MilacicHawRothfelsEtAl2012}, that rely on domain experts for curation and are under continuous development. This expert curation can often be a confounder when they are used in the variant set enrichment setting, as genes implicated through GWAS results are often used to augment disease specific pathways.  Furthermore, the complex, pleiotropic and hierarchical nature of the gene sets derived from such sources means that care needs to be taken not to double count genes that exist in multiple pathways when computing enrichment statistics. Another issue is the stability of gene sets  over time, as greater biological insight is gained, this can mean that significant results obtained with one version of the database might over time be eroded. Most fundamental is the fact that these resources cannot create novel biological insight beyond  the re-purposing of an existing pathway within a novel context, indeed one of the main utilities for such analysis is providing support for existing findings, as employed in Section \ref{sect:tp_reactome}.

Another source of gene sets is from differential analysis of high through put genomic data such as transcriptomics. In this context a specific hypothesis might be tested using either public or private data sets to see if differentially expressed genes are enriched for trait associations. Such approaches can reveal novel insights into underlying biology, for example, I have previously used transcription factor binding perturbation experiments to implicate specific factors that might have a role in type 1 diabetes~\citep{Burren2014-vh}. Finally a more hybrid approach shows promise where gene sets are derived from the overlap of carefully curated transcriptomics experiments that are grouped on the basis of broad functional classes, for example  the MSigDB Hallmark gene sets~\citep{Subramanian2005-ke}. These provide the basis of a more exploratory analysis overcoming some of the limitations previously mentioned.  



%We used a prior ($\pi_{i}$) of $10^{-4}$ which means that we expect 1 in 10,000 SNPs to be causal for a trait
\section{High resolution promoter capture Hi-C}
The most novel aspect of this project is using a data driven approach to link putative causal variation to the target genes in relevant tissues. Techniques such as Hi-C~\citep{Lieberman-AidenvanBerkumWilliamsEtAl2009} and  Chromatin Interaction Analysis by Paired-End Tag Sequencing (ChIA-PET)~\citep{FullwoodLiuPanEtAl2009} have been developed to map the genome-wide chromatin interactions in specific cell types~\citep{RaoHuntleyDurandEtAl2014}. Hi-C involves cross-linking genomic DNA with formaldehyde resulting in covalent links between spatially adjacent chromatin segments. This chromatin is then digested with a restriction enzyme and sticky ends are filled in with biotin labelled nucleotides. Ligation is then performed under dilute conditions which favours intra-molecular ligation events. The DNA is then purified and then sonically sheared and fragments are then enriched for biotinylated junctions, which then undergo paired end sequencing~\citep{vanBerkumLieberman-AidenWilliamsEtAl2010}. ChiA-PET again involves using formaldehyde to covalently link spatially adjacent chromatin. After sonification an immunoprecipitation step is used select a protein of interest (e.g. transcription factor) and the chromatin to which it is bound. Next biotin conjugated linker sequences are ligated to the free ends of the immunoprecipitated DNA. A further proximity based ligation takes place with enrichment of biotinylated linkers. Barcodes within the linker sequences are used to resolve chimeric intra and inter complex interactions. 

Whilst both are genome-wide methods for interrogating chromatin looping, CHiA-PET targets a specific protein, and therefore, compared to Hi-C, is unable to give an unbiased overview of all chromatin interactions within a given tissue context. However, Hi-C resolution is limited by two main factors. Firstly the protocol involves a restriction enzyme digest, usually \textit{Hind}III, and interactions are called based on the resultant fragments generated, in practice if \textit{Hind}III is used this limits resolution to approximately 4Kb. Secondly,  the complexity of the sequence libraries generated means that to increase the effective resolution of conventional Hi-C by a factor $n$ requires $n^{2}$ sequencing reads which is prohibitive for general implementation~\citep{JaegerMiglioriniHenrionEtAl2015}. Promoter capture Hi-C (PCHi-C) incorporates a sequence capture extension to classical Hi-C to enrich for chromatin interactions with protein coding gene promoters~\citep{Mifsud2015-fm}. Such PCHi-C allows for increasing resolution in an approximately linear fashion with increased sequence depth allowing for an economically viable approach for identifying promoter interactions. 

%\section{Aims}


%My main aim is to understand whether PCHi-C can provide a better understanding of how causal variation modulates autoimmune disease. For convenience this can be subdivided into answering three specific questions:

%\begin{enumerate}
%	\item Can PCHi-C help us to identify causal variation ?
%	\item What information can PCHi-C provide about genes and cellular contexts modulated by causal variants ?
%	\item Can these genes and cellular contexts identify novel pathways that inform the biology of autoimmunity ? 
%\end{enumerate}

%In this report I describe the development of a methodological framework that can be used to integrate tissue specific PCHi-C interaction maps with both targeted and genome wide genetic studies. Using a set of interaction maps for 17 haematopoietic primary Human cell types and focusing on autoimmunity I use this framework to define lists of prioritised variants, genes and cellular contexts. I show how these can be further refined by using  deep genomic phenotyping of  CD4$^{+}$ T cells  in activated and non-activated states. Finally I discuss followup experiments in \textit{IL2RA} that demonstrate context specific allelic imbalance, that provides validation of this framework.

\chapter{Materials and Methods}

\section{Promoter capture interaction maps of 17 haematopoetic primary human cells}
I obtained maps of 17 primary human cells of the haematopoetic lineage that were generated collaboratively with members of the Fraser, Spivakov, Ouwehand and Diabetes and Inflammation Laboratories. Each cell type was assessed over an average of 3 biological replicates (Appendix Table \ref{tab:pints}). The bespoke capture platform employed encompassed a total of 22,076 \textit{Hind}III fragments containing 31,253 annotated promoters for 18,202 protein-coding and 10,929 non-protein coding genes (Ensembl v75). Significantly interacting regions were called using the CHiCAGO pipeline~\citep{CairnsFreire-PritchettWingettEtAl2016}. Interactions with a CHiCAGO score threshold of $>$ 5 were used in all downstream analysis. These maps consist of a bait \textit{Hind}III fragment and a list of interacting \textit{HindIII} fragments or promoter interacting regions (PIR) within a particular cellular context.

\section{Collection and quality control of GWAS summary statistics from 31 genome wide association studies}
I downloaded GWAS summary statistics, covering 8 autoimmune and 23 other traits, from online resources, as detailed in Appendix Table \ref{tab:gwasm}. Genotyping error can create false associations,  therefore I filtered all association statistics to include only robust associations by removing those SNPs which were genome-wide significant ($p < 5 \times 10^{-8}$) but for which no variants, within 500Kb and in LD ($r^{2} > 0.6$) with the lead SNP, had $p < 1 \times 10^{-5}$. Finally I removed any SNP that was genome-wide significant but was not found in the 1000 Genomes PhaseIII EUR genotype set. 
%This code is found in gwas_integration/R/QC.R
\section{Poor Man's Imputation (PMI) - Imputation of GWAS p-values to the 1000 Genome reference panel in the absence of effect size and direction}
There was a high degree of variation in the coverage of GWAS summary statistics obtained, some studies contained information on approximately $5 \times 10^5$ variants where as others were imputed to 1000 genome reference genotype set and contained in excess of $7 \times 10^6$ variants. Imputation can be used to compute approximate association statistics for missing variants, however, it requires access to underlying genotype data, which in this case was unavailable for most traits. Methods exist for imputing summary statistics in the absence of genotyping data such as \textit{GCTA}~\citep{YangLeeGoddardEtAl2011} and \textit{IMPG}~\citep{Pasaniuc2014-im}, however these rely on access to either signed $Z$ scores, odds ratios or $\beta$ coefficients and their standard errors, in order to estimate direction of effect, which are not always available. I therefore designed an alternative `best guess' method, poor man's imputation (PMI), which requires evaluation but allows the processing of a wide range of traits for which variant coverage is heterogeneous and only univariate $p$ values are available.

The pipeline I developed, approximates the $p$-value for missing SNP summary statistics for a given study using a suitable reference genotype set. Firstly the genome is split into regions based on a recombination frequency of 0.1cM using HapMap recombination rate data. For each region we retrieve from the reference genotype set (1000 genomes EUR cohort) all SNPs that have MAF $ > 1\%$ and use these to compute pairwise LD. The pipeline pairs each SNP with missing $p$-values to the SNP with maximum pairwise $r^2$, ${r^2}_{max}$, if that ${r^2}_{max} > 0.6$, and impute the missing $p$-value as that at the paired SNP. SNPs with missing data or without a pair above threshold are discarded as are SNPs that are included in the study but do not map to the reference genotype set.

\section{Causal variant posterior probabilities for 31 traits using GWAS summary statistics}
To fine map candidate causal variants I used Wakefield's synthesis of approximate Bayes factors (ABF)~\citep{Wakefield2009} in connection with the method described in \citet{The_Wellcome_Trust_Case_Control_Consortium2012-ad} (see Section \ref{sect:finemapping}), to compute posterior probabilities for each SNP within 0.1 cM regions, using R code adapted from the \textit{coloc} package~\citep{GiambartolomeiVukcevicSchadtEtAl2014}.  

I set the value of prior of the any variant being causal ($\pi_{i}$) to that from \citet{GiambartolomeiVukcevicSchadtEtAl2014}, $10^{-4}$, which means that we expect 1 in 10,000 SNPs to be causal for a trait. This framework assumes a model where either no SNPs are causal within a region or that exactly one SNP is causal. I masked the MHC region (GRCh37:chr6:25-35Mb) from all downstream analysis due to its extended LD and known strong and complex association with autoimmune diseases

\section{Comparison of posterior probabilities between PMI and classical imputation}
Before using PMI posterior probabilities in further analyses it was necessary to assess whether they approximated posterior probabilities derived from classical imputation. Firstly I selected all SNPs mapping to Chromosome 1 as a representative sample. To create a simulated non-imputed data set I pruned these results to contain only SNPs for which p-values were reported in ~\citet{Stahl2010-wz}. I next ran PMI on this pruned data set and using \textit{bedtools}~\citep{Quinlan2014}, merged these with actual the imputed p-values from~\citet{Okada2014-um}. I confined my comparison to those SNPs imputed by PMI, a total of 235,412 SNPs

\
%\section{Variant posterior probabilities of inclusion using full genotyping data}

%For four diseases (autoimmune thyroid disease, celiac disease , rheumatoid arthritis and type 1 diabetes) we had access to full genotyping data from ImmunoChip. Chris Wallace carried out formal imputation to 1000 genomes project EUR using IMPUTE2 and fine mapped causal variants in each of the 179 regions where p$<$0.0001 was observed using a stochastic search method that allows for multiple causal variants within a region~\citep{WallaceCutlerPontikosEtAl2015}. 

\section{\textit{blockshifter} - A competitive test for associated variant enrichment in PCHi-C interaction maps}
In order to examine the enrichment of GWAS signals in promoter interacting regions (PIRs) I developed a method based on ideas implemented in \textit{GOSHIFTER}~\citep{Trynka2015-wz} to examine the enrichment of GWAS signals in the promoter interacting regions, in order to overcome both linkage disequilibrium (LD) and interaction fragment correlation. \textit{blockshifter} implements a competitive test of enrichment between a test set of PIRs compared to a control set. Firstly the coordinates of the PIRs in the union of test and control sets are retrieved, and PIRs with no overlapping GWAS signal  are discarded. As the test is competitive between PIRs in test and control sets, those overlapping both are not informative and therefore are excluded from further analysis. For the remaining PIRs we store the number and sum of overlapping GWAS posterior probabilities and these are used to compute $\delta$, the difference in the means of posterior probabilities between the test and control set. Due to correlation between GWAS signals and between PIRs the variance of $\delta$ is inflated, I therefore estimate it empirically using permutation.  Runs of one or more PIRs (separated by at most one \textit{Hind}III fragment) are combined into ‘blocks’, that are labelled unmixed (either test or control PIRs) or mixed (block contains both test and control PIRs).  Unmixed blocks are permuted in a standard fashion by reassigning either test or control labels randomly across blocks taking into account the number of blocks in the observed sets. Mixed blocks are permuted by effectively circularising each block and rotating the labels (figure \ref{fig:blockshifter}). I store the mean posterior probabilities across each possible permuted block. The number of choices at each block is small, but there are many blocks. I then randomly sample from each these precomputed block permutations $n$ times so that the proportion of underlying PIRs labels is the same as the observed set and use this to compute the set of $\delta_{null}$. I use $\delta_{null}$ to compute an empirical $Z$-score:
\begin{equation}
Z = \frac{\delta - \overline{\delta_{null}}}{\sqrt{V*}}
\end{equation}
Where $V^{*}$ is an empirical estimate of the variance of $\delta_{null}$. 

\begin{figure}[h]

\includegraphics[width=\textwidth]{blockshifter.pdf}
\caption{Permutation strategy for two `mixed superblocks' in \textit{blockshifter}. GWAS
summary statistics are converted to posterior probabilities for a given
SNP to be causal (Red SNPs are likely to be causal). Mixed superblocks
are runs of adjacent \textit{Hind}III PIRs  found in either test (purple) or
control (blue) tissue sets that are separated by two or more \textit{Hind}III
fragments for which no PIR in either set exists (grey). Posterior
probabilities can be assigned to test or control labels based on PIR
overlap and the sum for each category can be stored. The difference in
the weighted mean of the sum of posterior probabilities between test and
control sets can be used to compute enrichment.  However, to control for
inflation due to correlation structure between SNPs and between
interactions we rotate the labels of \textit{Hind}III fragments within the mixed
superblock to generate a set test and control posterior probabilities
under the null. Such a strategy is not required for unmixed superblocks
with a single label.  By sampling from these null test statistics across
the set of mixed and unmixed superblocks (weighted so that we select
similar numbers of test and control PIRs to the observed data set)  we
can rapidly generate an empirical null genome wide. These can be used to
adjust the test statistic to account for inflation due to underlying
correlation. Figure prepared with M. Spivakov. }
\label{fig:blockshifter}
\centering
\end{figure}

\section{COGS - An algorithm for PCHi-C assisted prioritisation of genes and tissues contexts}
\label{sect:COGS}
Whilst the \textit{blockshifter} method provides information about relevant tissue contexts, the primary goal of this project is to use the PCHi-C information to link causal variation to target gene(s). To do this
I developed an algorithm to compute tissue specific gene scores for each GWAS trait, taking into account linkage disequilibrium, interactions and functional SNP annotation (figure \ref{fig:cogs}). For each gene annotation, for which we have at least one significant interaction and for every nearby recombination block, the algorithm computes a block gene score that is composed of three components.

\begin{figure}[h]
\includegraphics[width=\textwidth]{cogs3.pdf}
\caption{A schematic illustration of the COGS (\textbf{C}apture Hi-C \textbf{O}mnibus \textbf{G}ene \textbf{S}core)  method. GWAS summary statistics are imputed by PMI and converted to posterior probabilities across three linkage disequilibrium (LD) blocks. These are then intersected with PCHi-C maps, promoter interacting regions and their linkages to bait \textit{Hind}III fragments allow the assignment of posterior probabilities to a focal gene (green). Due to limitations in the ability to call very short range interactions we create `Promoter' regions that consist of bait and adjoining \textit{Hind}III fragments. We account for protein coding SNPs by considering those in the focal gene (green) and masking others. Assuming independence between LD blocks we sum the information across categories to compute an overall gene score. In the above example naive analysis might prioritise the non focal gene (grey), however PCHi-C interactions provide additional information supporting the focal gene(green).}
\label{fig:cogs}
\centering
\end{figure}

\begin{enumerate}
\item The contribution due to coding SNPs in the annotated coding gene as computed by VEP~\citep{McLaren2010-ne}.
\item The contribution due to promoter SNPs, which I define as SNPs that overlap the bait and it's pair of flanking \textit{Hind}III fragments and not any coding SNPs.
\item The contribution due to SNPs that overlap interacting other ends for a tissue or set of tissues that do not overlap coding SNPs. 
\end{enumerate}

For a given target gene and recombination block, the algorithm derives a block gene score that is the sum of the posterior probabilities of SNPs overlapping each component, which is the probability that there exists a causal variant for this gene in this block, under the assumption that there is at most one causal variant in any block. Assuming independence we can combine blocks to get an overall gene score such that:

\begin{equation}
  Gene\ score = 1- \prod_j  \left(1-\sum_{i \in R_j} 1-PP_i  \right)
  \label{eqn:cogs_score}
\end{equation}

Here $PP_i$ is the posterior probability for the $i^{th}$ variant to be causal, $i \in R_j$ is the set of relevant SNPs $i$ in the $j^{th}$ region. Whilst components one and two are fixed for a given gene and trait, the contribution of variants overlapping PIRs varies, depending on the tissue context being examined. I developed a hierarchical heuristic method to ascertain for each target gene which was the mostly likely component and cell state. Firstly for each gene I compute the gene score due to genic effects (components 1 $+$ 2) and interactions (component 3) using all available tissue interactions for that gene. I use the ratio of gene effects score to interactions score in a similar manner to a Bayes factor to decide whether one is more likely. If gene effect is more likely (ratio $>$3) I iterate and compare if the gene score due to coding variants (component 1) is more likely than for promoter variants (component 2). Similarly if an interaction is more likely I compare interaction gene scores for one set of tissue(s) to another. If at any stage no branch is substantially preferred over its competitor (ratio of gene scores $<$ 3) I return the previous set as most likely, otherwise I continue until a single cell state/set is chosen. In this way I can prioritise genes based on the overall score and label a likely mechanism for candidate causal variants (Figure \ref{fig:AHR_dend}).

For the four traits where a stochastic search method was employed I adapted COGS to work with multiple models by aggregating the  posterior probabilities for each model with a variant overlapping the PIRs, promoters or coding variants to compute marginal posterior probabilities of inclusion.

\section{Comparison of COGS scores to non PCHi-C methods}
\label{sect:nonpchic-method}
To assess the utility of COGS scores and whether PCHi-C data sets were adding information, I generated comparative scores using two other methods orthogonal to PCHi-C. The intent behind this analysis was to assess the utility of interaction data, therefore, as input to all methods, I supplied PMI data sets from which I had masked coding variation and variation mapping to the MHC region on chromosome six. Firstly, I had access to topological associated domain (TAD) boundary information, supplied by Csilla Varnai, Michiel Thieke and Mikhail Spivakov for six cell types (Table \ref{tab:hic_coverage}).



For each TAD in each cell type, I subdivided and summed posterior probabilities for each trait (excluding the MHC region) by overlap with 0.1cM recombination blocks to obtain block TAD scores, and computed an overall TAD score such that:
 \begin{equation}
 \label{eqn:tad_score}
TAD.score = 1-\prod_{\text{blocks}}(1-TADscore.block).
\end{equation}
A TAD score was assigned to each gene mapping within the respective TAD in each tissue, and the maximum score across all eight tissues was selected

Secondly, I created gene scores based on proximity of associated variants to gene promoters. I took all \textit{Hind}III fragments within $\pm$ 0.5 Mb of a genes baited \textit{Hind}II fragment, and using 0.1cM recombination blocks computed proximity block scores, these were combined using a similar method detailed in equation \ref{eqn:tad_score}. Finally I computed a comparison COGS score using only PCHi-C maps for  tissues for which TAD boundary information were available (Table \ref{tab:hic_coverage}).

\begin{table}[ht]
\centering
\begin{tabular}{cc}
  \hline
 Tissue & TAD Coverage (Gb) \\ 
  \hline
Erythroblasts & 1.53 \\ 
  Macrophages & 1.68 \\ 
  Megakaryocytes & 1.59 \\ 
  Monocytes & 1.48 \\ 
  Naive B cells & 1.51 \\ 
  Naive CD4$^+$ T cells & 1.40 \\ 
  Naive CD8$^+$ T cells & 1.51 \\ 
  Neutrophils & 1.27 \\ 
   \hline
\end{tabular}

\caption{Topologically associated domain coverage across eight cell types elucidated from classical Hi-C analysis}
\label{tab:hic_coverage}
\end{table}

\section{Enrichment of COGS in disease specific differentially expressed genes}
As a naive method to assess the biological relevance of the genes prioritised by COGS, and to allow quantitative comparison between the TAD, Proximity and COGS methods, I examined  differentially expressed genes from \citet{PetersLyonsLeeEtAl2016} for enrichment of genes prioritised on the basis of each method. To do this I used differential expression analysis of this data set supplied by Chris Wallace. The data set consists of PEER normalised microarray expression values across 49 patients with Crohn's disease, 42 with ulcerative colitis and 43 healthy controls, across sorted  CD4$^+$ T cells, CD8$+$ T cells, B cells, Monocytes, and Neutrophils. Differential expression was computed with a null hypothesis that expression for a given gene was the same across all three groups within a tissue. As COGS and TAD scores are derived by combining over cell types I selected the tissue with the maximum significant differential expression values across tissue for a given gene. As the differential expression analysis concerned ulcerative colitis and Crohn's disease I used as input GWAS statistics from \citet{Anderson2011-ch} and \citet{Franke2010-mj} with coding and MHC variants removed.

\section{Reactome Pathway Analysis}
Using modified R code developed by Mikhail Spivakov, for each trait I selected all protein coding genes having an overall gene score above 0.5. I converted Ensembl gene identifiers to Entrez gene identifiers using bioMaRT~\citep{DurinckSpellmanBirneyEtAl2009} and used ReactomePA~\citep{YuHe2016} to compute the enrichment of genes within the Reactome pathways using an FDR cutoff of 0.05, using ClusterProfiler~\citep{YuWangHanEtAl2012} to plot a bubble plot of significant results.  

\section{Causal variant posterior probabilities  using ImmunoChip summary statistics}
Dense genotyping platforms target specific genomic regions to provide higher resolution genetic maps. I wanted to see, in the context of activated and non activated CD4$^{+}$ T cells, what extra information could be gained by integration of PCHi-C maps with genetic data from such targeted studies.  Focusing on Autoimmune traits,  I downloaded  ImmunoChip summary statistics for six traits from \url{http://www.immunobase.org} carrying out QC as previously described. The ImmunoChip is a targeted genotyping platform for dense coverage of approximately 180 regions with robust demonstration of association with one or more autoimmune traits~\citep{CortesBrown2011}. Summary statistics for ulcerative colitis, Crohn's disease and psoriasis were supplied privately by study authors. I fine mapped these traits using the PMI method previously described, but replacing 0.1cM regions with the 179 regions (median size 227Kb with an inter quartile range of between 126Kb and 392Kb) that were densely genotyped on the ImmunoChip~\citep{Onengut-Gumuscu2015-lb}. 

\section{Comparison of PMI with genotype method allowing for multiple causal variants in a region}

To evaluate the effect on COGS scores of the PMI  assumption of a single causal variant I used a set of marginal posterior probabilities obtained by Chris Wallace using a stochastic search method, GUESSFM, that allows for multiple causal variants within a region~\citep{WallaceCutlerPontikosEtAl2015}. I compared  four diseases(autoimmune thyroid disease, celiac disease , rheumatoid arthritis and type 1 diabetes)   for which we had access for full genotyping data from ImmunoChip and summary statistics. I incorporated these into further analysis on the assumption that full genotyping data and imputation would provide more accurate posterior probabilities at the targeted regions.



%Whilst components 1 and 2 are fixed for a given gene and trait the contribution of SNPs overlapping PIRs varies depending on the tissue context being examined. We developed a hierarchical heuristic method based on the Haematopoietic tree to ascertain for each target gene which was the mostly likely component and tissue context. Firstly for each gene we compute the gene score due to gene effects (components 1 + 2) and interactions (component 3) using all available tissue interactions for that gene. We use the ratio of gene effects score to interactions score in a similar manner to a Bayes factor to decide whether one is more likely. If gene effect is more likely we iterate and compare if the gene score due to coding SNPs (component 1) is more likely than for promoter SNPs (component 2). Similarly if an interaction is more likely we begin descending the Hematopoietic tree and compare interaction gene scores for sets of tissues (e.g Myeloid vs Lymphoid). If at any stage the algorithm is unable to choose it stops and return the previous set as most likely, otherwise it continues until a single tissue/set is chosen. In this way we can prioritize genes based on the overall score and label as to a likely mechanism for candidate causal variants.




%\section{Tissue specific HALLMARK gene set enrichment analysis}
%Using modified R code developed by Chris Wallace  I used Wilcoxon rank sum tests to compare the distribution of gene prioritisation scores for genes in each MSigDB HALLMARK~\citep{LiberzonBirgerThorvaldsdottirEtAl2015} set to its complement, within the set of genes that both had membership of at least one HALLMARK set and had a gene score.  I used the $p$ value from this test, together with the difference in mean gene prioritisation scores, to generate a signed $Z$ score, $Z_{ig}$ for trait $i$ and gene set $g$.  To test for relative enrichment in autoimmune diseases versus non-autoimmune traits, we used $t$ tests to compare the distributions of $Z_{ig}$ for $i$ indexing autoimmune diseases to the $Z_{jg}$ for $j$ indexing non-autoimmune traits.

%\section{IL2RA allelic imbalance}

%Move to the discussion
%Targetted sequencing of \textit{IL2RA} in CD4$^{+}$ T cells was carried out, by Daniel Rainbow, across four donors that were heterozygous at eight group A SNPs and homozygous for group C and F SNPs (Figure\ref{fig:il2ra_ase}). This was carried out at 0, 2 and 4 hours after stimulation. Data processing was carried out using the Methpup package ~\citep{RainbowYangBurrenEtAl2015} (https://github.com/ollyburren/Methpup) to extract counts of each allele at rs61839660. Chris Wallace carried out the statistical analysis of the results. Briefly, the fraction of A2/(A1+A2) was compared across replicates in each cDNA sample to gDNA within individuals using Wilcoxon rank sum tests.  Where there was a bias in the same direction in each individual, combined p values were generated across individuals using Fisher’s product of $p$-values method.   


\chapter{Results}

\section{Posterior probabilities generated by PMI and genotype imputation are comparable}

To validate performance of the PMI technique, I compared the results as imputed by PMI to those as reported by  ~\citep{Okada2014-um}. There was good agreement between PMI imputed $- \log_{10} p$-values and those derived from classical imputation as reported in~\citet{Okada2014-um} ($\rho=0.9418103$, figure \ref{fig:pmi_comparison}). % need to add something about why they are sometimes different and how this relates to MAF ?

\begin{figure}[h]
\centering
\includegraphics[width=0.7\textwidth]{pmi_vs_imp.pdf}
\caption{Comparison of $P$-values imputed by PMI versus those reported in ~\citep{Okada2014-um} for chromosome. Axis are $-\log_{10}$ transformed $p$-values}
\label{fig:pmi_comparison}
\end{figure}

%\subsection{Generation of Promoter Capture Hi-C interaction maps across 17 primary human cell types}
%Promoter Capture Hi-C interaction maps for 17 primary human cell types were generated as part of a collaboration between the Diabetes and Inflammation Laboratory(DIL), Fraser and Spivakov Groups and the BLUEPRINT consortium. Significant interactions were called using CHiCAGO~\citep{CairnsFreire-PritchettWingettEtAl2016} (Table\ref{tab:pints}) and these were then reannotated to EnsEMBL 75 using \textit{bedtools}~\citep{Quinlan2014}. To enable visual inspection of these results alongside GWAS summary statistics we developed the CHiCP~\citep{SchofieldCarverAchuthanEtAl2016} (https://www.chicp.org) webserver. 

%\subsection{Fine mapping of  GWAS summary statistics across 31 traits}
%We created a compendium of GWAS summary statistics from a vareity of sources for 31 traits (Table\ref{tab:gwasm}). In order to maximise resolution and coverage we imputed to 1000 Genome Phase III reference genotype set ~\citep{1000_Genomes_Project_Consortium2015-wc}. Compare PMI to Imputed. 
%Many studies were missing direction of effect statistics precluding the use of existing summary statistics imputation methods such as  \textit{ImpG}~\citep{Pasaniuc2014-im}. We therefore developed an LD based approach, which we named Poor Man's Imputation (PMI).We used Wakefield's~\citep{Wakefield2009} synthesis of approximate Bayes factors to compute posterior probabilities for a single variant being causal, within 0.1cM recombination blocks. We masked the MHC region (GRCh37:chr6:25-35Mb) from all downstream analysis due to its extended LD and known strong and complex association with autoimmune diseases.

\section{Tissue specific enrichment of associated variants with PIRs across 31 traits}
Enrichment of GWAS signals in tissue specific enhancers has been previously described~\citep{MauranoHumbertRynesEtAl2012}, and we expect PIRs to be enriched for regulatory regions. As such a demonstration of robust enrichment, within tissue specific PIRs, is a pre-requisite for further analysis that attempts to link causal variation to target genes in relevant tissue contexts.  Using \textit{blockshifter} with the PMI imputed summary statistics from the 31 GWAS assembled (Table \ref{tab:gwasm}), I found that variants associated with autoimmune disease are enriched in PIRs in lymphoid compared to myeloid tissues (Figure \ref{fig:bs_1}). In contrast, SNPs associated with erythroid traits, including mean haemoglobin concentration (MCH), mean corpuscular volume (MCV) and red blood cell count (RBC) showed a selective enrichment in erythroblasts and megakaryocytes compared to PIRs in monocytes, macrophages and neutrophils (Figure \ref{fig:bs_1}). I next examined whether I could further resolve tissue differences using \textit{blockshifter}. I found that autoimmune traits were enriched in activated and non-activated CD4$^{+}$ T cells when compared to megakaryocytes and erythroblasts. This enrichment for autoimmune disease traits was stronger in activated  compared to non activated CD4$^{+}$ T cells (Figure \ref{fig:bs_2}).


\begin{figure}[h]
\centering
\includegraphics[width=0.8\textwidth]{block_shifter_scatter.pdf}
\caption{Enrichment of GWAS summary statistics at PIRs by tissue groups. Axes reflect \textit{blockshifter} $Z$-scores for two different tissue group comparisons, firstly lymphoid versus myeloid and then within myeloid lineage (MON - Monocyte, MAC - Macrophage and NEU - Neutrophil) versus (MEG - Megakaryocyte and ERY - erythroblast). Traits are labelled and coloured by category (Appendix Table \ref{tab:gwasm}).}
\label{fig:bs_1}
\end{figure}

\begin{figure}[h]
\centering
\includegraphics[width=0.8\textwidth]{tcell_blockshifter_scatter.pdf}
\caption{Tissue specific enrichment of autoimmune GWAS summary statistics at PIRs by tissue groups. Axes reflect \textit{blockshifter} $Z$-scores for two different tissue group comparisons, firstly MK - megakaryocytes and ERY - erythroblast versus Activated/Non-activated CD4$^{+}$ T cells. Y-axis shows comparison of Activated versus non activated CD4$^{+}$ T cells. Autoimmune traits are coloured red and other traits are blue, point size reflects the log transformed sample number included in each study.}
\label{fig:bs_2}
\end{figure}

\section{PCHi-C assisted gene prioritisation across 31 traits}
\label{sect:tp_reactome}
Using COGS, I prioritised 2,604  unique protein coding genes with an overall COGS score greater than 0.5, across all 31 traits examined, with a median of 122 genes prioritised per trait (Figure \ref{fig:31_trait_counts}). The mean number of protein coding genes skipped was 1.5 for genes prioritised based on promoter interactions. The prioritised genes exhibited enrichment for specific pathways in the Reactome pathway database~\citep{FabregatSidiropoulosGarapatiEtAl2016} in a naive analysis. As expected, genes prioritised for autoimmune diseases were enriched in inflammation and immune response-related pathways, such as interleukin and T cell receptor signalling, whereas genes prioritised for platelet traits were preferentially associated with platelet production and haemostasis (Figure \ref{fig:reactome}). 

\begin{landscape}

\begin{figure}[ht]
\centering
\includegraphics[]{all_trait_gene_counts.pdf}
\caption{Prioritised gene counts for 31 traits. Colours indicate the proportion of genes in four broad hypotheses of causal variation mechanisms : Multiple - No specific hypothesis favoured; Coding - cSNP(s) in target gene; Promoter - cSNP(s) around TSS of target gene; Interaction - cSNP(s) in PIR of target gene.}
\label{fig:31_trait_counts}
\end{figure}

\end{landscape}

\begin{landscape}
\begin{figure}[p]
\centering
\includegraphics[scale=0.5]{hyper_reactome_analysis_2.pdf}
\caption{Bubble plot of traits with significant enrichment ($P_{adj} < 0.05$) in one or more reactome pathways  for genes with COGS score $>$ 0.5 across 31 traits. Number in parentheses below trait labels indicate the total number of genes analysed for each trait, bubble size indicates the ratio of test genes to those in the pathway, and blue to red shading corresponds to decreasing adjusted $P$-value for enrichment}
\label{fig:reactome}
\end{figure}
\end{landscape}



%In addition, the genes prioritised for RA included 5/9 candidates (C8Orf13, BLK, TRAF1, FADS2 and SYNGR1) that were identified in a recent study (Zhu et al., 2016) combining whole-blood eQTL with RA GWAS data by Mendelian randomisation. The relatively large number of prioritized genes without eQTL support is in agreement with previous reports of limited overlap of disease variants with eQTLs (Guo et al., 2015; Huang et al., 2015). This demonstrates complementary benefits of eQTL-based and physical interaction-based prioritization approaches for prioritizing identifying candidate target genes of non-coding disease variants.


%\section{Context specific gene priortisation across activated CD4$^{+}$ activated T cells}


\section{Prioritised genes do not overlap significantly with eQTLs}
In parallel with my GWAS focused analysis Oliver Stegle and Roman Kretzhuber (RK) had been working to consider expression quantitative trail loci (eQTLs) from \citet{FairfaxMakinoRadhakrishnanEtAl2012} in the context of PCHi-C. Both eQTL and GWAS are enriched in PIRs, which prompted us to consider the overlap of COGS prioritised genes in SLE~\citep{Bentham2015-di} and rheumatoid arthritis~\citep{Okada2014-um} data sets, for which full imputed summary statistics were available. I performed COGS gene prioritisation and RK looked to see if there was evidence for overlap with relevant tissue eQTLs~\citep{FairfaxMakinoRadhakrishnanEtAl2012}.  Out of 456 genes that were prioritised for both traits 136 had eQTLs  of which four genes (\textit{BLK}, \textit{RASGRP1}, \textit{SUOX}, and \textit{GIN1}) showed evidence for possible co-localisation in RA and two genes (\textit{BLK} and \textit{SLC15A4}) in SLE. Additionally the genes prioritised for RA included 5/9 candidates (\textit{C8Orf13}, \textit{BLK}, \textit{TRAF1}, \textit{FADS2} and \textit{SYNGR1}) that were identified in a recent study~\citep{ZhuZhangHuEtAl2016} that combined whole blood eQTLs with the same RA GWAS data by Mendelian randomisation. The relatively large number of GWAS prioritised genes without eQTL support  agrees with previous reports of limited overlap of disease variants with eQTLs~\citep{Guo2015-ka,HuangChenEsparzaEtAl2015}. One reason for this observation might be that  causal variants function in a specific tissue context, which has yet to be suitably assessed by eQTL analysis. As suggested by~\citet{Huang2015-ug}, this implies that regulatory annotations that might be used to augment fine mapping methods are similarly incomplete. 

\section{COGS prioritisation qualitatively performs better than TAD and distance based scores}
For all eight GWAS autoimmune traits, I compared protein coding genes scores, using PCHi-C data sets (COGS), and comparative scores computed using either TAD or proximal methods. To summarise results across all diseases I selected the maximum score for a method for each gene. Using a score cut off of 0.5 I then categorised each gene as to whether it was prioritised in either a single, both or no methods (Figure \ref{fig:cog_prox_comparison}). In general COGS prioritised genes sets were significantly smaller than then those from the other methods. The naive proximity based score appeared to be least specific prioritising 5,731 genes in total, however it had the greatest overlap with COGS scores implying comparatively better sensitivity. For proximity based scores 83 genes were prioritised exclusively by COGS, indicating that 12\% of genes are prioritised by interactions greater than 0.5 Mb. For TAD based scores 219 genes were prioritised exclusively by COGS, indicating that approximately 30\% of genes are prioritised by interactions that span TAD boundaries.  I note however that examining distance metrics between gene baits and supporting PIRs that in some cases this could be due to the imprecise definition of TAD boundaries (Figure \ref{fig:distance_distro}). Overall COGS appears to select genes not found by other methods and also appears to be more specific.

\begin{figure}[h]
\centering
	 \begin{subfigure}{0.7\textwidth}
		%\includegraphics[width=\linewidth]{dot.pdf}
		\includegraphics[width=\linewidth]{prox_compare.pdf}
		\caption{}\label{fig:fig_a}
	\end{subfigure}
	\hspace{1cm}
	\begin{subfigure}{0.7\textwidth}
		\includegraphics[width=\linewidth]{tad_compare.pdf}
		\caption{}\label{fig:fig_b}
	\end{subfigure}
\begin{minipage}[t]{0.7\textwidth}
	\caption{Comparison of autoimmune PCHi-C COGS scores and (\subref{fig:fig_a}) a proximity score from assigning variants to genes within 0.5 Mb of gene promoters (\subref{fig:fig_b}) Hi-C derived TAD scores, using seven Cell types (Erythroblasts, Macrophages, Monocytes, Naive B cells, Naive CD4$^{+}$ T cells, Naive CD8$^{+}$ T cells and Neutrophils). Counts of genes in each quadrant are shown, grey to red colour gradient indicates gene density.}
	\label{fig:cog_prox_comparison}
\end{minipage}
\end{figure} 

\begin{figure}[ht]
\centering
\includegraphics[width=0.5\textwidth]{dist_boxplot.pdf}
\caption{Box plot showing the distribution of distances between baits and TAD boundaries for significant (score $>$ 0.5) genes. ’Both’ indicates that gene was significant using TAD and COGS methods}
\label{fig:distance_distro}
\end{figure}

\section{COGS prioritised genes are enriched for differentially expressed IBD genes} 
Calibrating performance metrics of the COGS method is challenging in the absence of a set of functionally validated set of scores. One possible proxy is to examine the overlap of prioritised genes with studies examining differential expression between diseased and healthy volunteers in relevant tissue types. I integrated scores from PCHi-C (COGS), proximity and TAD methods, for ulcerative colitis (UC) and Crohn's disease (CD), with differentially expressed genes (FDR 5\%) from~\citet{PetersLyonsLeeEtAl2016} and used Fisher's test to examine enrichment for prioritised genes (score $>$ 0.5). I found that UC COGS genes were significantly enriched for differentially expressed genes ($P = 0.002$), for Crohn's disease there was nominal enrichment ($P = 0.04$). Conversely there was no evidence of enrichment in any data sets using proximal and TAD methods (Figure \ref{fig:peters_odds_ratio}). The intersect between differentially expressed genes and COGS prioritised genes identified 67 genes (Table \ref{tab:ibd_genes}). Whilst there are many candidate genes in this list, an example on chromosome 3 for ulcerative colitis provides an illustration of the potential for hypothesis generation by integrative analysis of PCHi-C interaction maps (Figure \ref{fig:bcl6}). COGS prioritises \textit{BCL6}  which has been shown to have potent effects on Th9 cell development and IL-9 secretion both important modulators of inflammation~\citep{BassilOrentOlahEtAl2014}. Putative causal variants from~\citet{Anderson2011-ch} for UC reside in intron 8 of \textit{LPP}, and previous studies have implicated a causal role for this gene, however PCHi-C results from lymphoid tissues show interactions between this region and the \textit{BCL6} promoter (Figure \ref{fig:bcl6}). Furthermore, BLUEPRINT chromatin state maps classify this region harbouring putative causal variants as having enhancer activity across 3 out of 4 biological replicates in CD4$^+$ T cells. Expression profiles from~\citet{PetersLyonsLeeEtAl2016} show that whilst \textit{LPP} is not significantly differentially expressed between UC controls and healthy volunteers ($\log(\text{Fold Change})$ = 0.05, $P_{adj}$ =  0.46) \textit{BCL6} is ($\log(\text{Fold Change})$ = 0.22, $P_{adj}$ =  0.0018), providing further support for it's prioritisation.

\begin{figure}[ht]
\centering
\includegraphics[width=0.7\textwidth]{CHiCP-BCL6-Naive_CD4.pdf}
\caption{Circularised plot showing overlap between ulcerative colitis (UC) putative variants and  interactions between \textit{BCL6} gene promoter in CD4$^+$ T cells, prioritised by the COGS algorithm. A) Interactions, grey (not significant i.e. CHiCAGO score $>$ 5 in other tissues), blue (less significant) to red(more significant) colour gradient. B) UC GWAS summary $-\log(P)$ values from~\citet{Anderson2011-ch} C)  Gene annotations, protein coding (blue), antisense (purple), lincRNA (orange) and \textit{BCL6} (green). Red ticks around circumference indicate \textit{Hind}III fragment boundaries. Figure modified from CHiCP - \url{http://www.chicp.org}~\citep{SchofieldCarverAchuthanEtAl2016} }
\label{fig:bcl6}
\end{figure}

\begin{figure}[ht]
\centering
\includegraphics[width=0.7\textwidth]{ibd-tadprox.pdf}
\caption{Enrichment of prioritised genes in ulcerative colitis and Crohn's differentially expressed genes from~\citet{PetersLyonsLeeEtAl2016}. Methods PCHi-C (COGS), $<$ 500 Kb (Proximity) and within TAD (topologically associated domain).}
\label{fig:peters_odds_ratio}
\end{figure}

\section{Tissue specific PCHi-C assisted gene prioritisation using dense summary statistics}
The stronger enrichment of autoimmune disease GWAS traits in activated CD4$^+$ T cells lead me to examine autoimmune traits in the context of activated and non activated CD4$^{+}$ promoter interaction maps. I extended COGS to  incorporate a simple binary decision tree model to allow tissue or functional category resolution (see Section \ref{sect:COGS}).  Across the combined GWAS and ImmunoChip autoimmune data sets I was able prioritise 602 distinct protein coding genes  (Figure \ref{fig:cogs_ai}). To summarise the behaviour of prioritisation, I focused on a subset of 220 input autoimmune GWAS regions with genome-wide significant signals ($P<5 \times 10^{-8}$). I prioritised at least one gene with a COGS score $>0.5$ in 122 of these regions, with a median of two genes/region (inter quartile range = 1-3). The average distance from peak signal to prioritised genes was 334 kb, and I observed a median of three genes `skipped' between GWAS signal peaks and prioritised genes. Using pooled total RNA-seq expression data on the same donors (Anthony Cutler, Arcadio Rubio Garcia and Chris Wallace), I found that 457 of the prioritised genes were expressed in at least one activation state. I could relate 259 genes to GWAS significant signals of which 166 were deferentially expressed between activated and non activated states.

\section{Allowing multiple causal variants increases number of prioritised genes}
I wanted to understand the effect of assuming a single causal variant within each region on a genes COGS scores for various traits. I compared COGS scores computed from marginal posterior probabilities for dense genotype data from four autoimmune diseases (ATD, CEL, RA and T1D) , computed by GUESS FM, which allow multiple causal variants to those from PMI, which assumes a single common variant (figure \ref{fig:gfm_vs_pmi_gs}). Generally there was agreement between methods, however I note that i GUESSFM prioritises more genes than PMI (Table \ref{tab:cogs_gfm_pmi}). This increase in specificity may be down to increased resolution gained from the GUESSFM approach due to the employment of genotype based imputation and it's ability to consider different prior probabilities. However, some of this will be offset by a greater sensitivity of GUESSFM to genotyping error than the PMI based approach.

\begin{table}[ht]
\centering
\begin{tabular}{lrrrr}
  \hline
Disease & GUESSFM & PMI & Both & Total \\ 
  \hline
ATD &   9 &   0 &   6 &  38 \\ 
CEL &   6 &   5 &  33 & 114 \\
RA &   6 &   2 &  19 & 132 \\ 
T1D &  16 &   7 &  35 & 212 \\ 
\hline
\end{tabular}

\caption{Counts for protein coding genes prioritised (score $>$ 0.5) by GUESSFM, Poor Man's Imputation (PMI) and Both methods out of Total genes with score $>$ 0.01.} 
\label{tab:cogs_gfm_pmi}
\end{table} 

%\section{Gene set enrichment analysis of COGS prioritised genes using MSigDB HALLMARK gene sets} 

%To check whether the list of prioritised genes corresponded to existing biological understanding of autoimmune disease, in collaboration with Chris Wallace we used gene set enrichment analysis of the genome wide COGS scores as described in the methods  across 31 traits to MSigDB HALLMARK gene sets. We found significantly greater (Bonferroni $p<0.05$) enrichment in autoimmune diseases compared to non-autoimmune traits in four gene sets relating to signalling and receptor molecules as well as reactive oxygen species involved in early T cell activation (figure \ref{fig:hallmark}).

\begin{figure}[h]
\centering
\includegraphics[width=0.7\textwidth]{ai_cogs.pdf}
\caption{Functional gene prioritisation across 11 autoimmune diseases using genome wide (GWAS) or targeted genotyping array (ImmunoChip) data. The numbers at each node give the number of genes prioritised at that level. Where there is evidence to split into one of two non-overlapping hypotheses ($\log_{10}$ ratio of gene scores $>$3), the genes cascade down the tree. Where the evidence does not confidently predict which of the the two possibilities is more likely, genes are `stuck' at the parent node (number given in brackets). When the same gene is prioritised for multiple ($n>1$) disease, I assign a fractional count to each node, defined as the proportion of the $n$ diseases for which the gene was prioritised at that node.}
\label{fig:cogs_ai}
\end{figure}

%\begin{figure}[h]
%\includegraphics[width=\textwidth]{hallmark_enrichment.pdf}
%\caption{Tissue specific gene set enrichment analysis using COGS gene prioritisation scores for 11 autoimmune traits. Colours correspond to shared pathways enriched in modules identified by weighted gene correlation network %analysis (WGCNA) of activated vs non-activated CD4$^{+}$ T cells RNA-Seq data. Points give the difference in average enrichment of $Z$ scores across autoimmune versus non-autoimmune GWAS and bars the 95$\%$ confidence interval. Significant gene sets after Bonferroni correction are indicated by $*$.}
%\label{fig:hallmark} 
%\end{figure}

\begin{figure}[h]
\includegraphics[width=\textwidth]{gfm_vs_pmi_gs.pdf}
\caption{Comparison of COGS gene scores for four autoimmune traits (ATD = autoimmune thyroid disease, CEL = celiac disease, RA = rheumatoid arthritis and T1D = type 1 diabetes computed using marginal posteriors from GUESSFM and those computed using posteriors from PMI, grey line shows $y=x$, blue lines indicate the fitting of a linear model}
\label{fig:gfm_vs_pmi_gs}
\end{figure}


%\section{CHiCP: a genome browser for PCHiC interaction maps}
%In collaboration with Ellen Schofield, I developed a web based application, CHiCP \url{https://www.chicp.org}, to allow the visual integration of both public and private promoter capture Hi-C maps with GWAS and BLUEPRINT epigenetic data~\citep{SchofieldCarverAchuthanEtAl2016}. To do this I developed an interactive circularised display, with a search interface  for genes, variants and chromosomal coordinates. An interaction can be selected, allowing users to access a conventional browser view to see a zoomed in view of chromatin states and GWAS signals at promoter and promoter interacting regions.

\chapter{Discussion}

I have presented novel methods for the integration of three dimensional chromatin data with GWAS summary stats in order to prioritise genes and target tissues. A competitive enrichment analysis, \textit{blockshifter}, that carefully accounts for correlation, provided evidence that such a strategy was justified, implicating CD${^+}$ T cell activation as an important tissue context, in the development of autoimmune diseases.  COGS, a Bayesian framework for gene  prioritisation, enabled a data driven approach to gene prioritisation, to complement current approaches.  Comparisons of COGS prioritised genes with those derived from TAD and proximity based methods,  show that COGS has greater sensitivity and specificity providing support for the general approach.  In Section \ref{section:il2ra} I describe one such targeted experimental approach to validate one of the findings. However, collaboratively we have developed a reciprocal capture Hi-C platform to perform targeted validation of 949 specific PIRs identified from PCHi-C approach. Such a platform can  help to overcome some technical limitations of PCHi-C. Firstly, its more targeted nature allows greater read-depth and therefore more sensitive detection of less stable or prevalent chromatin interactions. Secondly, it is not confined to promoter interactions and will allow interrogation of the inter-PIR chromatin interaction space. Indeed preliminary analysis has indicated the presence of strong interactions between the PIR detailed in section \ref{section:il2ra} and another PIR in a separate but proximal autoimmune region containing PRKCQ (COGS score=0.95)~\citep{LoweCooperBruskoEtAl2007}. 

A fundamental tenet of science is reproducibility, and whilst technical considerations inject variability at the level of data generation, considerable efforts should be made to develop transparent analytical methods that do not add to this. The software that I have developed over the course of this project is available \url{https://github.com/ollyburren/CHIGP}, however, due to the organic nature of its development its overall usability and architecture is lacking. To foster community reuse I intend to rewrite the software into a formal R package, complete with regression testing and more extensive documentation. I hope that this initial time investment will not only benefit the community but also provide a solid foundation for the future work that I set out below. 

%This prioritsation allows a data driven approach to the design of followup validation experiments, allowing more precise targeting of trait/disease specific interactions, genes and tissue contexts.

\begin{figure}[ht]
\centering
\includegraphics[width=0.8\textwidth]{AHR_dend.pdf}
\caption{COGS decision dendrogram using rheumatoid arthritis data from \citet{Okada2014-um} and PCHi-C maps from 17 haematopoetic cell types for the \textit{AHR} gene. The red edges denote the path taken by COGS through the binary decision tree. Each node is labelled with pseudo Bayes factors(pBF). Terminal nodes are labelled with the tissue/annotation specific gene score(GS) }
\label{fig:AHR_dend}
\end{figure}

\begin{figure}[ht]
\centering
\includegraphics[width=0.85\textwidth]{RA_dend.pdf}
\caption{COGS decision dendrogram for all prioritised protein coding genes (GS $> 0.5$) using rheumatoid arthritis data from \cite{Okada2014-um} and PCHi-C maps from 17 haematopoetic cell types. Edges are coloured based on number of genes flowing between connected nodes. Nodes are marked with the total number of genes at a node and in brackets the number of genes that are assigned to that node.}
\label{fig:RA_dend}
\end{figure}

 
%\section{Summary} %prerhaps goes at the end ?
%Methods to integrate genetic and genomic data sets are important in order to identify causal variants, genes and pathways and the tissue contexts within which they operate. I have developed two Bayesian approaches to integrate GWAS summary statistics and targeted genotyping data (ImmunoChip) with high resolution chromatin conformation data. Firstly \textit{blockshifter}, a method to perform competitive tissue enrichment analysis in the presence of correlation in order to prioritise relevant tissue contexts. Secondly COGS, a method to prioritise causal candidate genes based across tissue contexts and traits.  I used \textit{blockshifter} to verify the relevance of chromatin interactions with activated and unactivated CD4$^{+}$ T cells to autoimmune traits. Using COGS I was able to prioritise a number of genes for further study, I performed gene set analysis using a standard hypergeometric method on the Reactome database and a tissue specific GSEA using MolSigDB HALLMARK gene sets which indicated that COGS prioritised genes  were enriched for both expected and unexpected pathways. 

\section{Functional validation of COGS prioritised gene \textit{IL2RA}}
\label{section:il2ra}
I have presented \textit{in silico} validation for genes prioritised by integrating PCHi-C and GWAS data sets, by the application of pathway and gene set enrichment analysis, however, empirical evidence is required to fully validate the strategy. Thus, in collaboration with Daniel Rainbow, Tony Cutler, Arcadio Rubio Garcia, Chris Wallace and Linda Wicker we sought to understand how we might functionally validate one such prioritised gene and whether this might lead to a greater understanding of how causal variation might modulate autoimmune disease susceptibility. COGS analysis prioritised, in multiple diseases (autoimmune thyroid disease, Crohn's disease, multiple sclerosis, rheumatoid arthritis, type 1 diabetes, and ulcerative colitis) \textit{IL2RA} which encodes the CD25 protein,  a component of the IL-2 receptor that is essential for high-affinity binding of IL-2, regulatory T cell survival and T effector cell differentiation and function~\citep{LiaoLinLeonard2013}. I found this prioritisation to be driven by an interaction between the \textit{IL2RA} promoter and a PIR in exon 1 known to harbour a set of type 1 diabetes putative causal SNPs (Figure \ref{fig:il2ra_ase_region}) identified in a previous fine mapping study~\citep{WallaceCutlerPontikosEtAl2015}. This  set of SNPs (Figure \ref{fig:il2ra_ase_region}) is in high LD ($r^{2} > 0.8$) with rs12722495 which has been shown to affect the surface expression CD25 in memory T cells~\citep{DendrouPlagnolFungEtAl2009}. Using a targeted RNA-sequencing approach, and software I helped to develop previously~\citep{RainbowYangBurrenEtAl2015}, Daniel Rainbow measured the relative expression of the two alleles at one of these SNPs, rs61839660, in intronic cDNA, from four individuals heterozygous at rs61839660 and homozygous across most other associated SNP groups, in a four-hour activation time-course of CD4$^{+}$ T cells. We observed allelic imbalance in non activated CD4$^{+}$ T cells however on activation this was lost suggesting a context specific effect within this locus. To ensure this reflected imbalance in transcription of \textit{IL2RA}, and not simply enhancer RNAs in intron 1,  further validation was performed using rs12244380 found in the 3' UTR of \textit{IL2RA} (Figure \ref{fig:il2ra_ase_tc}). 

\begin{figure}[h]
\centering
\includegraphics[width=\textwidth]{il2ra_ase_region.pdf}
\caption{PCHi-C interactions link the \textit{IL2RA} promoter to autoimmune disease associated genetic variation which leads to expression differences in \textit{IL2RA} mRNA. (n) and (a) refer to non activated and activated CD4$^+$ T cells respectively. Numbers in parentheses on the right hand side represent number of interactions observed. Labels `A' to `F' indicate sets of SNPs likely to contain a single causal disease variant.  Figure prepared by Tony Cutler, Arcadio Rubio Garcia and Chris Wallace.}
\label{fig:il2ra_ase_region}
\end{figure}

\begin{figure}[h]
\centering
\includegraphics[width=0.6\textwidth]{il2ra_ase_tc.pdf}
\caption{\textit{IL2RA} allele specific expression in CD4$^{+}$ T cells for two putative causal variants identified by fine mapping and PCHi-C integration. Bar colour encodes DNA collected under different conditions. Black indicates genomic DNA, we expect this to be equally shared between alleles. Blue indicates cDNA collected a time zero. Green and purple bars indicate cDNA allele read counts collected under non activated and activated conditions. Figure prepared by Chris Wallace.}
\label{fig:il2ra_ase_tc}
\end{figure}

\section{Limitations}
%write about heterogenous cell populations and the effect that this could have.
There are a number of limitations with the current approaches that I have developed. Firstly the fine mapping approach employed assumes for a given region a simplified model of a single underlying causal variant. I demonstrate that this does have an effect on gene prioritisation but further work is required to whether the genes prioritised using GUESSFM are in fact better candidates for functional follow up.

Another limitation is the thresholded approach to CHiCAGO scores that are used to call interactions. All of the methods developed so far use a threshold score of 5, so that an interaction with a score of 4.99 will be omitted. Future approaches might investigate methods for incorporating promoter interaction scores themselves in both \textit{blockshifter} and COGS methods. One approach might be to look at techniques that utilise CHiCAGO scores across multiple tissues for a given interaction to adjust local FDR. 

COGS makes the assumption that coding variation affects the gene within which it is located, however studies in model organisms~\citep{LawrieMesserHershbergEtAl2013} and in humans~\citep{StergachisHaugenShaferEtAl2013} indicate that coding variation can fulfil a dual role in the regulation of genes. Whilst coding putative causal variants are relatively rare it remains to be seen what effect this assumption has on overall COGS scores, I hope to investigate this further using the approach detailed in Section \ref{sect:gwassim}.

Another significant challenge with these analysis is that resolution is limited by \textit{Hind}III restriction fragment length. This manifests in two main ways. Firstly there is a blind spot for observing shorter range interactions that involve \textit{Hind}III fragments and adjoining baited interactions. I have attempted to capture these as `promoter' regions (Figure \ref{fig:cogs}), however integration with functional annotation might provide further resolution and identify functional hypothesis for mechanisms for further study. 

A second more pernicious issue is that many baited fragments are promiscuous in that they contain promoter regions for more than one gene. If we consider just protein coding genes then of 16,608 baits, 3,009 ($18\%$) contain multiple promoters from different genes. If I include all transcriptional start site annotations in Ensembl (Version 75) then this rises to 6,703 ($40\%$). In reality, even this is a conservative estimate due to the incomplete annotation of the non-coding genome. For these promiscuous baits it is impossible to resolve which promoter or promoters are involved in the chromatin looping, using the current PCHI-C data alone. One approach is to use gene expression patterns to provide a filter for target genes identified by specific chromatin interactions, which I discuss in section \ref{sect:mo_celltypes}. Furthermore, my analysis to date has concentrated on protein coding genes as these have the most mature and complete annotation, however,  recent work has suggested a role for non coding genes in the modulation of autoimmune disease susceptibility ~\citep{Castellanos-RubioFernandez-JimenezKratchmarovEtAl2016}. The PCHi-C platform used does provide some coverage of the non protein coding genome, however this is not exhaustive by design and due to the overlapping nature of the coding and non-coding genome, the issue of promiscuous baits is exacerbated.

Excluding single cell implementations, all genomic technologies give an average of the molecular events across the (sometimes mixed) population of cells being assayed. In the case of immune subsets this is particularly relevant as broad categories, such as CD4$^{+}$ T cells will be heterogeneous containing further subdivisions that may or may not be relevant for disease biology. It is important to bear this in mind when considering PCHi-C maps as without single cell profiling it is impossible to resolve whether interactions are common across the assayed tissue type or are specific to an underlying and as yet unsorted subset.



\chapter{Future Work}

\section{Creation of a catalogue of putative causal variants and genes across 17 cell types and 31 traits}
\label{sect:mo_celltypes}

Enrichment results from \textit{blockshifter} coupled with existing results in the literature  lead to a focus on understanding the tissue specificity of PCHi-C interactions within activated and non-activated CD4$^{+}$ T cells. The next step is to generate tissue specific COGS scores for all GWAS data sets across all 17 cell types. By way of example it should be possible to generate decision trees for all prioritised genes (e.g. Figure \ref{fig:AHR_dend} ), enabling a deeper understanding of the distribution of tissue contexts on a disease wide scale (Figure \ref{fig:RA_dend}). Further analysis will delve into the details of why a particular gene is targeted, in order to select candidates that are suitable for functional followup. Criteria might include,  posterior probability concentrated on a single SNP, with an interaction observed in a specific cellular context for which relevant gene expression is observed (e.g \textit{BCL6}). Such analysis would include further investigation of methods for more formal integration of gene expression and TAD data by, for example, taking a more tissue specific approach using the ~\citep{PetersLyonsLeeEtAl2016}  data set. I would bolster this analysis by incorporating new data from the targeted validation platform, which might shed light on relevant PIR to PIR interactions which cannot be captured in the primary data set.

\section{Genomic annotation assisted fine mapping} 

 Increasingly high quality genomic data sets on primary human tissues are becoming available. I have shown that both non activated and activated CD4$^{+}$ T cells are of particular interest and  have access to Chip-Seq and total RNA-Seq for these tissues. Future work will explore whether these annotations can be used as input to integrative methods such as \textit{fgwas} to increase fine mapping resolution. I will start with autoimmune data sets for which I have access to GWAS summary statistics and use \textit{fgwas} to integrate functional annotations, concentrating on in house ChIP-Seq and ATAC-Seq data sets available for CD4$^{+}$ T cells. A concern with \textit{fgwas} is that it relies on extensive cross validation to overcome it's reliance on a single training set, to overcome this I will use the enrichment parameters generated from application to GWAS and use these to compute variable prior probabilities for dense fine mapping information using equations \ref{eqn:fgwas_var_prior} and \ref{eqn:fgwas_var_prior}. These  prior probabilities can be used to compute `annotation aware' posterior probabilities for a variant to be causal. I will use these as input into COGS to understand if and how this alters the genes prioritised. Depending on the outcome of this it might be worth examining RiVIERA~\citep{LiKellis2016} that can make use the covariance across related traits.

% need to state what datasets we might use.

\section{Data driven discovery of appropriate COGS score thresholds}
\label{sect:gwassim}
Understanding how fine scale genetic architecture affects COGS scores is key to their interpretation, for example, ascertaining a suitable threshold for gene prioritisation. One approach requires curation of a number of non coding,  causal variants with convincing functional validation for a target gene and tissue context. Unfortunately such examples are rare and so investigation of the effect of parameters such as effect size and minor allele frequency on COGS score is limited. One possible solution is to accurately simulate GWAS based on predefined parameters and causal variants and to use this as the input to assess COGS scores at specific intervals or genome wide. Using software developed by Mary Fortune that allows the accurate simulation of GWAS statistics for a given interval I will generate  summary statistics to allow me to understand how various parameters affect the specificity and sensitivity of COGS and any extensions I develop.

\section{Using COGS scores for gene set enrichment analysis}
A future aim of this project is to investigate whether we can use information gained on individual genes to highlight relevant biological pathways in a tissue context specific manner. I have used the Reactome database to validate genes prioritised by COGS score, however as discussed in the introduction such analysis are imperfect. Future work will investigate two approaches using tissue specific PCHi-C, in the context of gene set enrichment analysis (GSEA). In the first approach I will extend the naive approach used to analyse~\citet{PetersLyonsLeeEtAl2016} data set to use tissue specific COGS scores, a concern with this approach is the spatial correlation introduced by COGS scores and whether GWAS simulation methods I will investigate in section \ref{sect:gwassim} can be used to account for this. The second approach will attempt to adapt the multivariate normal SNP permutation method implemented in VSEAMS~\citep{Burren2014-vh}, to use PCHi-C maps in order to assign SNPs to genes. One challenge will be to allow for the LD based SNP pruning  required for computational efficiency without undermining the resolution of PCHi-C maps. As both methods will examine the problem through the prism of tissue context specificity, I will need to consider effective normalisation methods in order to accurately assess the results. One approach is to focus on autoimmune traits and use the set of 23 non autoimmune traits to assess the relative enrichment between both sets. I will assess the output of this against an extension of the TAD method described in section \ref{sect:nonpchic-method}. Depending on the outcome of this I can extend by using the GWAS simulation method \ref{sect:gwassim} in order to generate sets of GWAS statistics with plausible parameters to assess enrichment. This will allow the assessment of traits without the need to normalise by trait category. 




% Need to add more detail as to how Mary's software works and what parameters I intend to look at 

%One possible naive solution is to integrate context specific expression data to match genes expressed in that context to possible context specific interactions. This leads on to another possible extension which is to extend prioritisation to include annotated non coding genes as a recent study suggests that these modulate autoimmune disease susceptibility~\citep{Castellanos-RubioFernandez-JimenezKratchmarovEtAl2016}.

%\section{Application of COGS scores to the full PCHiC dataset}
%Having shown evidence that the COGS tissue specific prioritisation score is useful, I have begun the process of applying it across all traits to all 17 tissue types. For each trait it should be possible to generate a decision tree as in Figure \ref{fig:AHR_dend}. If this is successful I would like to examine metrics for comparing tree structures and therefore inter trait relationships (Figure \ref{fig:RA_dend}).





%We should perhaps do this - kind of done - slightly worried about the missing data.
%There are a number of assumptions that are made for these data that require further investigation of their validity. Firstly for traits where only summary GWAS statistics are available we assume a single causal variant for an LD block, one outstanding question is to what effect this has on COGS scores. I aim to do this by comparing traits where we have summary statistics and genotype data and comparing COGS scores of genes between PMI and GUESSFM methods. 

%\section{Comparison with Hi-C defined TAD boundaries}
%So far it is unclear as to how much information is gained in gene prioritisation using promoter capture Hi-C over classical Hi-C. Csilla Varnai and Michiel Thiecke have generated TAD domain boundaries for 8 of the 17 cell types assayed using conventional Hi-C libraries. I will use these to generate pseudo interaction matrices whereby within each TAD the promoter of all genes interacts with every fragment, excepting the baited fragment, and those bounding it for a given gene within a TAD. For these tissues for which Hi-C libraries are available I will also compute gene scores using COGS and analyse the differences to see whether information is gained from using promoter-capture Hi-C over classical Hi-C. 


%\section{Integrating other annotations into COGS}

%PMI assumes a fixed prior for all SNPs however recent work~\citep{Pickrell2014-xs} using Bayesian hierarchical frameworks demonstrates a method by which we can estimate variable priors for each SNP based on other sources of annotation. In the hierarchical method implemented by \textit{fgwas} for a given SNP the prior probability for association, $\pi_{ik}$, varies depending on the enrichment of annotations, $\lambda_{l}$. 

%\begin{equation}
%	\pi_{ik} = \frac{e^{x_{i}}}{\sum_{j \in S_k}e^{x_{j}}}
%\end{equation}

%$x_i$ is the sum of the effect of all the annotations that the $i^{th}$ SNP overlaps as shown below.  Here $\lambda{i}$ is the effect of annotation $l$ and $I_{il}$ is and indicator function as to whether SNP $i$ overlaps annotation $l$.  

%\begin{equation}
%	x_{i} = \sum_{l=1}^{L_{2}} \lambda_{l}I_{il}
%\end{equation}

%Initially I would use further information on $CD4^{+}$ T cells including various histone-modification marks and methylation data alongside public ally available annotations to compute $\lambda_{l}$ using \textit{fgwas}. Using a model including the most relevant annotations these $x_{i}$ values could also be applied to compute variable priors ($\pi_{i}$) for dense ImmunoChip summary statistics. These more annotation aware posterior probabilities will then be input into COGS to see how this alters gene prioritisation. If this was successful I would look into the incorporation of other BLUEPRINT annotations across all 31 traits.
 
 %what about BLOCKSHIFTER ? If we add variable priors is this good or bad ? I guess in a non efficient way it gives you conditional enrichment ?  

% \section{blockshifter development - perhaps omit}
 
%Demonstrating the enrichment of an annotation for associated variants robustly in the presence of correlation is challenging. Correlation, occurring through linkage disequilibrium between SNPs and between genomic annotations,  has the effect of inflating the underlying text statistic and must be carefully adjusted for. The \textit{blockshifter} approach does this using a mixture of circularised and weighted permutation techniques using underlying \textit{Hind}III fragments as atomic units to define underlying block structure. I will investigate the utility of \textit{blockshifter} outside of the PCHiC initially looking at it's performance using (un)activated CD4$^{+}$ T cell ChIP-Seq data sets. In this case I am not limited to \textit{Hind}III but will assay the performance of more frequent cutters (e.g. \textit{SspI}) in terms of results and computational efficiency.
%Can the RF assumptions employed for BS be used as a more general purpose enrichment tool ?
%For COGS can we use fgwas to create variable priors to help increase the resolution ? Also not a big fan of CHiCAGO thresholding approach is there a way we can extend to include interaction probabilities ?



%\section{Future Directions}
%In the longer term I hope to collaborate with clinicians to build promoter capture Hi-C maps of relevant tissues in disease and healthy states. Controlling for variation between individuals in this context is paramount, thus employing strategies to overcome this within the bounds of the economic and technical limitations is important. One strategy is to compare diseased and healthy tissue within the same individual, taken at the same time point to  look for differences in interactions, an example of this might be in juvenile idiopathic arthritis (JIA), here the diseased tissue is known and samples, in the form of synovial fluid drains for are available. PCHi-C results generated from CD4$+$ T cells isolated from the same individual might be compared with those isolated from the periphery. Another complimentary strategy for systemic autoimmune disease where diseased tissue is challenging to collect is to take a longitudinal approach, here patients presenting with a disease are assayed, with further follow up assays collected over time taking into account any therapeutic interventions. Genomic comparison between time points is then conducted to look for differential interactions that might be related to disease prognosis. Further evidence of for a genes involvement in autoimmune disease susceptibility might come from adapting the methods previously described to work with rare variants. For this work one could use gene level variant aggregation tests~\citep{LeeTeslovichBoehnkeEtAl2013} and then use genotype data from primary immune disorders collected as part of the BRIDGE study, PCHiC data would be used to assign variants to genes allowing the integration of both coding and non-coding variation.

\renewcommand{\abstractname}{Acknowledgements}
\begin{abstract}
Thanks to: Chris Wallace, John Todd and Mikhail Spivakov for supervision; Tony Cutler and Linda Wicker for assistance in interpreting immunological relevance of results; Ellen Schofield, Prem Acuthan and Tim Carver for ImmunoBase and CHiCP support and development; Mary Fortune for statistical advice; Other members of  the DIL, Wallace, Spivakov, Fraser, Ouwehand and Stegle Labs for data access and analytical support.
\end{abstract}

\appendix

	\include{pchic_interaction_count_table}
	\begin{landscape}
	 	\include{gwas_meta_table}
	\end{landscape}
	\begin{landscape}
	\include{ichip_meta_table}
	\end{landscape}
	\include{ibd_gene_list}
\bibliographystyle{genomeresearch}
%\bibliographystyle{plain}
\bibliography{first_year_report}

\end{document}
